pkg/packagekit/history.jsx

```diff
@@ -29,22 +29,34 @@ import cockpit from "cockpit";
 
 const _ = cockpit.gettext;
 
-function formatPkgs(pkgs) {
-    const names = Object.keys(pkgs).filter(i => i != "_time");
-    names.sort();
-    return names.map(n => {
-        const tooltipRef = React.useRef(null);
+const PackageItem = ({ name, value }) => {
+    const tooltipRef = React.useRef(null);
+
+    return (
+        <React.Fragment>
+            <li ref={tooltipRef}>{name}</li>
+            <Tooltip triggerRef={tooltipRef} content={`${name} ${value}`} />
+        </React.Fragment>
+    );
+};
 
-        return (
-            <React.Fragment key={n}>
-                <li ref={tooltipRef}>{n}</li>
-                <Tooltip triggerRef={tooltipRef} content={ n + " " + pkgs[n] } />
-            </React.Fragment>
-        );
-    });
-}
+export const PackageList = ({ packages }) => {
+    if (!packages) {
+        return null;
+    }
```

Comment:
Lets improve the code while we're at it. Using [React fragments](https://react.dev/reference/react/Fragment) would be better so that `PackageList` always returns a `ReactNode`
```suggestion
    if (!packages) {
        return <></>;
    }
```